local-cli/generator-android/templates/src/app/build.gradle

```diff
@@ -83,13 +83,13 @@ def enableSeparateBuildPerCPUArchitecture = false
 def enableProguardInReleaseBuilds = false
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 24
+    buildToolsVersion "24.0.2"
 
     defaultConfig {
         applicationId "<%= package %>"
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 24
```

Comment:
Changing this will crash apps when a permission is denied by the user. Actually none of the inbuilt modules like geolocation are asking the permission for location. So this will likely break a lot of things.